### PR TITLE
us8 implemented, needs a code review

### PR DESCRIPTION
**Pull request by ghtina**
please review(sorry about the false name)

### CODE REVIEW

File: plugins/de/opendiabetes/vault/plugin/importer/vaultodv/VaultODVImporter.java [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+            super(new VaultCSVValidator(), ',');
+        }
+
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public boolean importData() {
+            String importPath = getImportFilePath();
+                    preprocessingIfNeeded(importPath);
```
Comment (lucasbuschlinger) [inline plugins/de/opendiabetes/vault/plugin/importer/vaultodv/VaultODVImporter.java]: Please use correct indentation.

File: plugins/de/opendiabetes/vault/plugin/importer/vaultodv/VaultODVImporter.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        }
+
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        protected List<VaultEntry> parseEntry(final CsvReader csvReader) throws UnsupportedOperationException {
+            throw new UnsupportedOperationException("ParseEntry not implemented for VaultODVImporter plugin! (Not needed");
+        }
+
```
Comment (lucasbuschlinger) [inline plugins/de/opendiabetes/vault/plugin/importer/vaultodv/VaultODVImporter.java:162]: Would write real JavaDoc to explain why it is implemented like this, not inherit it.

File: plugins/de/opendiabetes/vault/plugin/exporter/vaultodv/VaultODVExporter.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package de.opendiabetes.vault.plugin.exporter.vaultodv;
+
+/**
```
Comment (lucasbuschlinger) [inline plugins/de/opendiabetes/vault/plugin/exporter/vaultodv/VaultODVExporter.java:3]: Please adhere to JavaDoc standards.

File: test/de/opendiabetes/tests/plugin/importer/VaultODVImporterTest.java [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+import org.pf4j.DefaultPluginManager;
+import org.pf4j.PluginException;
+import org.pf4j.PluginManager;
+
+import java.nio.file.Paths;
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+
+/**
+ * Tests for the de.opendiabetes.vault.plugin.importer.vaultodv.VaultODVImporter plugin.
```
Comment (lucasbuschlinger) [inline test/de/opendiabetes/tests/plugin/importer/VaultODVImporterTest.java]: No need for package hierarchy here.

**Review by lucasbuschlinger — CHANGES_REQUESTED** [review]
- [x] Werden korrekte Typen benutzt?
- [x] Methoden haben korrekte Sichtbarkeit
- [x] Variablen haben korrekte Sichtbarkeit
- [x] Keine Rechtschreibfehler (Schrift/Grammatik)
- [x] Code gut verständlich
- [x] Eindeutige Variablennamen
- [x] Dokumentation in Form von JavaDoc vorhanden
- [x] Keine logischen Fehler
- [x] Checkstyle keine Fehler
- [x] FindBugs keine (drastischen) Fehler

Handle stream closing correctly (see findbugs) and see my comments, otherwise fine.

File: plugins/de/opendiabetes/vault/plugin/exporter/vaultodv/VaultODVExporter.java (hunk 3 above)
Comment (ghtina) [reply plugins/de/opendiabetes/vault/plugin/exporter/vaultodv/VaultODVExporter.java:3]: did it

**Comment by ghtina** [comment]
did it all

**Review by lucasbuschlinger — APPROVED** [review]

**Comment by lucasbuschlinger** [comment]
Well done, closing #10 because of this.

**Pull request merged**